website/docs/r/nrql_alert_condition.html.markdown

```diff
@@ -101,6 +101,7 @@ The following arguments are supported:
 The `nrql` block supports the following arguments:
 
 - `query` - (Required) The NRQL query to execute for the condition.
+- `data_account_id` - (Optional) **BETA PREVIEW: the `data_account_id` field is in limited release and only enabled for preview on a per-account basis.** The account ID to use for the alert condition's query as specified in the the `query` field. If `data_account_id` is not specified, then the condition's query will be evaluated against the `account_id`. Note that the `account_id` must have read privileges for the `data_account_id` or else the condition will be invalid.
```

Comment:
📝 Calling out here that it's in "limited release"; after we GA the field, we'll update the `Description`.